examples/python_mnist/model/requirements.txt

```diff
@@ -0,0 +1,4 @@
+scikit-learn==0.20.2
+numpy==1.16.2
+keras
```

Comment:
Fix Keras and TensorFlow versions, so this demo won't break after TF 2.0 release. This can happen since TF 2.0 is not backwards compatible with 1.0